src/autoComplete/autoCompleteListItem.tsx

```diff
@@ -27,14 +23,16 @@ export default class AutoCompleteListItem extends Component <AutoCompleteListIte
   render() {
     return (
       <ItemChoice
+        href={<button />}
+        narrow
         status={this.props.status}
-        selected={this.props.highlighted}
-        className={cc(['kirk-autoComplete-item', this.props.className])}
+        className={this.props.className}
+        label={this.props.item.title}
+        subLabel={this.props.item.description}
+        leftAddon={this.props.item.icon}
         onMouseDown={this.onMouseDown}
         onDoneAnimationEnd={this.props.onDoneAnimationEnd}
-      >
-        {this.props.children}
```

Comment:
What was the usage of children before?

Reply:
Well, we had the "inside" of the ItemChoice pass in `children`, now it's all handled by the `<Item>`